python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_sensors.py

```diff
@@ -267,6 +339,37 @@ def test_stop_sensor(self, graphql_context):
             == InstigatorStatus.STOPPED.value
         )
 
+    def test_set_cursor(self, graphql_context):
```

Comment:
for extra credit, add a test case on an automatically on sensor with no rows in the DB yet